common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java

```diff
@@ -30,6 +30,10 @@ public GoofyAPI(FiguraLuaRuntime runtime) {
         this.owner = runtime.owner;
     }
 
+    public void canLog() {
```

Comment:
there is a void in my soul only filled by booleans
```suggestion
    public boolean canLog() {
```

Reply:
ahead of you